### PR TITLE
🤏 Allow Coordinate Search

### LINKED ISSUE #104: Allow searching for coordinates
**Describe the the feature request**

The current Falling Fruit website allows the user to search for a location via coordinates. We currently don't have this feature. We need to adapt the search bar so that it can take coordinates as a possible search result. This may involve doing some additional parsing of the input/handling a special case. However, it's possible that Google Autocomplete already handles it. In that case, you should try to defer to Google if possible.

### PR DESCRIPTION
**Pull request by code-arman**
## Status:
:rocket: Ready

## Description

Fixes #104 

## Screenshots
Screen Shot 2021-11-01 at 10 58 42 PM

### CODE REVIEW

**Comment by ezwelty** [comment]
FYI I get this error in the preview as soon as I complete typing in two decimal numbers separated by a comma:

> TypeError: e.charCodeAt is not a function
    at le (reach-combobox.esm.js:991)
    at reach-combobox.esm.js:582
    at oa (react-dom.production.min.js:157)
    at Da (react-dom.production.min.js:176)
    at Gs (react-dom.production.min.js:271)
    at Cu (react-dom.production.min.js:250)
    at _u (react-dom.production.min.js:250)
    at Eu (react-dom.production.min.js:250)
    at gu (react-dom.production.min.js:243)
    at react-dom.production.min.js:123

**Comment by code-arman** [comment]
> FYI I get this error in the preview as soon as I complete typing in two decimal numbers separated by a comma:
> 
> > TypeError: e.charCodeAt is not a function
> > at le (reach-combobox.esm.js:991)
> > at reach-combobox.esm.js:582
> > at oa (react-dom.production.min.js:157)
> > at Da (react-dom.production.min.js:176)
> > at Gs (react-dom.production.min.js:271)
> > at Cu (react-dom.production.min.js:250)
> > at _u (react-dom.production.min.js:250)
> > at Eu (react-dom.production.min.js:250)
> > at gu (react-dom.production.min.js:243)
> > at react-dom.production.min.js:123

Yup, still in progress 😅

Have a fix for that issue locally now tho :)

File: src/components/search/Search.js [hunk 1]
```diff
@@ -94,16 +95,43 @@ const Search = (props) => {
...
-        await getPlaceBounds(descriptionToPlaceId.current[description]),
-      ),
+  const handleCoordinateSearch = () => {
+    const coordinate = new CoordinateParser(value)
+
+    return (
+      <ComboboxOption
+        as={SearchEntry}
+        key={`${coordinate.getLatitude()}, ${coordinate.getLongitude()}`}
+        value={`${coordinate.getLatitude()}, ${coordinate.getLongitude()}`}
```
Comment (Qrtn) [inline src/components/search/Search.js:113]: You can define a variable for this string, and just use it a couple times.

Comment (Qrtn) [reply src/components/search/Search.js:113]: You can make this a function `getCoordinatesResult()`, and just return null if the coordinate parser returns an error.

**Review by Qrtn — APPROVED** [review]
Looks great! Good work.

**Review by SirajChokshi — APPROVED** [review]
Looks good! Left a few comments for style

**Pull request merged**